lib/themes/dosomething/paraneue_dosomething/templates/system/patterns/tile.tpl.php

```diff
@@ -20,7 +20,7 @@
 <article class="tile <?php print $classes; ?>">
 
   <?php if ($content['status']): ?>
-  <a class="wrapper" href="/<?php print $content['link']; ?>">
```

Comment:
Leaving in that slash breaks the URL. The url() function builds a url that can be put directly into the href.